home/pi/auto_run.sh

```diff
@@ -180,4 +178,4 @@ echo "***********************************************************************"
 echo ""
 echo ""
 sleep 2
-tail -f /var/log/mycroft-speech-client.log
+mycroft-cli-client
```

Comment:
I'm nervous about launching in to this automatically.  Currently, the CLI loads logs into memory and never curates.  So it will grow indefinitely, which isn't great for a headless setup.